indico_toolkit/metrics/usermetrics.py

```diff
@@ -0,0 +1,92 @@
+import pandas as pd
+from typing import List, Dict
+from indico import IndicoClient
+from datetime import datetime
+from indico_toolkit.indico_wrapper import IndicoWrapper
+from indico_toolkit import ToolkitInputError
+from indico.queries.usermetrics import GetUserSnapshots
+from indico.filters import UserMetricsFilter
+
+
+class UserMetrics(IndicoWrapper):
+    def __init__(self, client: IndicoClient):
+        self.client = client
+
+    def get_user_metrics_df(
```

Comment:
See notes about refactoring below. This function might be better a property functiony like: `user_metrics.dataframe`

Reply:
Added to class attributes here - resolved - https://github.com/IndicoDataSolutions/Indico-Solutions-Toolkit/blob/e49e5e55a1c12d9a696330b5f1355051839ed5b2/indico_toolkit/metrics/usermetrics.py#L47